src/Exception.php

```diff
@@ -199,8 +199,6 @@ public function getCustomExceptionTitle(): string
     /**
      * Set Custom Translator adapter.
      *
-     * @param ITranslatorAdapter|null $adapter
-     *
      * @return $this
```

Comment:
no more need for `@return $this` comment I guess.

Reply:
`self` means `self`, `$this` means "instance of `static`". I would love to change this to `: static`, but it is supported as of PHP 8.0 :D

Last week I did quite large research on this for maximum compatiility and we should not add `: self` to classes where refactoring of child classes need to be supported.

Many IDEs prefer the return type in the code and does not hint any methods/properties from the child classes even if the phpdoc is presented. Just to let you know and due this I withholded changes of object return types in data, especially for `Model` which need to support full refactorability for child classes.

Reply:
OK, I see.